<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StackedWidget</class>
 <widget class="QStackedWidget" name="StackedWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="toolTipDuration">
   <number>3</number>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(98, 150, 187);</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="FRONT">
   <property name="windowOpacity">
    <double>1.000000000000000</double>
   </property>
   <property name="toolTipDuration">
    <number>3</number>
   </property>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>320</y>
      <width>261</width>
      <height>76</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 40pt &quot;montserrat&quot;;
color: rgb(85, 87, 83);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="SCAN">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>501</width>
      <height>371</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 40pt &quot;montserrat&quot;;</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>-2</number>
    </property>
    <property name="text">
     <string>Please Scan Your Boarding Pass...</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>190</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>next</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="SUCCESS">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>236</width>
      <height>68</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 40pt &quot;Cantarell&quot;;</string>
    </property>
    <property name="text">
     <string>SUCCESS!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>244</y>
      <width>375</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;Cantarell&quot;;</string>
    </property>
    <property name="text">
     <string>Retrieving Your Flight Information...</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="next2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>next</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="INFO">
   <widget class="QPushButton" name="go_somewhere">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>157</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Take Me Somewhere...</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gate:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>9</y>
      <width>43</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>9</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Flight:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>312</x>
      <y>9</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Depature Time:</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="WHERE">
   <widget class="QPushButton" name="gate">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Gate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toilet">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Toilets</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shop">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Shop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="food">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>350</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Food</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>WHERE TO????</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="SHOPS">
   <widget class="QPushButton" name="whs_shop">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>WHSmith</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next_shop">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="FOOD">
   <widget class="QPushButton" name="mcd_food">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Mcdonalds</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bburr_food">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Bar Burrito</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="CONFIRM_DEST">
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>You want to go to....</string>
    </property>
   </widget>
   <widget class="QLabel" name="location">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>LOCATION</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Is this correct?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yes_go">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>YES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="not_correct">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>NO</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page"/>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>slot1()</slot>
 </slots>
</ui>
